core/src/test/scala/stryker4s/mutants/applymutants/MatchBuilderTest.scala

```diff
@@ -154,6 +154,7 @@ class MatchBuilderTest extends Stryker4sSuite with TreeEquality {
     val topStatement = source.find(origStatement).value.topStatement()
     val mutant = mutants
       .map(m => topStatement transformOnce { case orig if orig.isEqual(origStatement) => m } get)
+      .filterNot(_.isInAnnotation)
```

Comment:
If annotations are already excluded in the `MutantMatcher`, is this still needed?

Reply:
Good point, it's actually not needed.